.changeset/chilly-cherries-build.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/ag-grid-theme": patch
+---
+
+Remove override style for pinned column's text
```

Comment:
Can you put the reasoning behind this was done in the changeset please, as well as make the changeset past tense.